### DISCUSSION TITLE
Setting priors on media ROAS

### DISCUSSION THREAD

**Discussion by robcurtis123** [sibylhe/mmm_stan#31] [Q&A]
Hello! Let's say I ran a separate experiment and learned that the ROAS on one of my media channels was around 3. How would I incorporate this prior into the MMM model? 

Currently in the code the priors for the media coefficients is normal(0,1).  Should I adjust this part?  If so, how would I adjust it so the prior for the ROAS is, say, normal(3,1)?

Thank you!

**Comment by sibylhe** [comment]
Yes for additive model.
In multiplicative model, convert it to percentage. Media coefficients in multiplicative model stand for how many % increase in y (sales) when x increases 1%. 
E.g., sales: y = 50000, media spend: x = 1000
1% of x is 10, ROAS is 3, increase of sales delta_y = 3 * 10 = 30
beta = 30/50000 * 100 = 0.06
So set its beta ~ normal(0.06, 0.02), 0.02 could be any standard deviation you prefer.

**Reply by robcurtis123** [reply]
Thanks for your response. What do the 50000 for sales and 1000 media spend represent, given that there are different spend and sales counts on different days.  Is this the average for the period?

**Reply by robcurtis123** [reply]
Also, if the variables are scaled by value/mean (as in your blog), how does that get factored into the equation?

**Reply by sibylhe** [reply]
Here are some questions that might help you think.
1. Average of which period, 1 year, 3 years? Does ROAS remain at a stable level of 3 in your selected period?
At what spend level ROAS is 3? Note that spend has a diminishing return.
2. 1% change in (x/x_mean) will lead to how many % change in (y/y_mean)?
If you are unable to decide, just test all your tentative solutions:)
